src/main/java/com/newrelic/graphql/mapper/GraphQLInputMapper.java

```diff
@@ -75,19 +75,22 @@ private JavaType getType(GraphQLType type) throws ClassNotFoundException {
         return mapper.getTypeFactory().constructCollectionType(List.class, innerType);
       }
     } else if (type instanceof GraphQLInputObjectType) {
-      return mapper.getTypeFactory().constructType(classInPackage(packageName, type));
+      return mapper
+          .getTypeFactory()
+          .constructType(classInPackage(packageName, ((GraphQLInputObjectType) type).getName()));
     } else if (type instanceof GraphQLEnumType) {
-      return mapper.getTypeFactory().constructType(classInPackage(packageName, type));
+      return mapper
+          .getTypeFactory()
+          .constructType(classInPackage(packageName, ((GraphQLEnumType) type).getName()));
     } else if (type instanceof GraphQLNonNull) {
       return getType(((GraphQLNonNull) type).getWrappedType());
     }
 
     return null;
   }
 
-  private Class<?> classInPackage(String packageName, GraphQLType type)
-      throws ClassNotFoundException {
-    String className = String.format("%s.%s", packageName, type.getName());
+  private Class<?> classInPackage(String packageName, String name) throws ClassNotFoundException {
```

Comment:
I see why we need the change here with the base types that have shuffled. Fortunately, there's a `GraphQLNamedType` interface we could take here so we don't have to spread out the `getName()` calls. Still have to make the cast at call sites, but seems cleaner that way. What do you think?